app/views/flood_risk_engine/enrollments/steps/_grid_reference.html.erb

```diff
@@ -4,75 +4,72 @@
   <%= form_group_and_validation(form, :grid_reference) do %>
     <%= f.label :grid_reference, class: "form-label" do %>
       <%= t(".grid_reference.form_label") %>
-      <span class="form-hint">
+      <span class="form-hint text">
         <%= t(".grid_reference.example_hint") %>
       </span>
-      <span class="form-hint">
+      <span class="form-hint text">
         <%= t(".grid_reference.clarification_hint") %>
       </span>
     <% end %>
     <%=
       f.text_field(
-        :grid_reference,
-        class: "form-control form-control-char-25",
-        "aria-describedby" => "ngr_help"
+          :grid_reference,
+          class: "form-control form-control-char-25",
+          "aria-describedby" => "ngr_help"
       )
     %>
   <% end %>
 
-  <!-- Find grid reference -->
   <details style="padding-bottom: 18px;" id="ngr_help">
     <summary><span class="summary"><%= t(".details.summary") %></span></summary>
     <div class="panel panel-border-narrow">
       <p>
         <%=
-          t(
-            ".details.tool_text",
-            link: link_to(
-              t(".details.tool_link"),
-              "https://www.ordnancesurvey.co.uk/osmaps/",
-              rel: "external",
-              target: "_blank"
-            )
+          t(".details.tool_text", link: link_to( t(".details.tool_link"),
+                                                 "https://www.ordnancesurvey.co.uk/osmaps/",
+                                                 rel: "external", target: "_blank")
           ).html_safe
         %>
       </p>
-      <p><%= t(".details.tool_info")  %></p>
+      <ul class="list list-number">
+        <li>
+          <%= t(".details.bullet1") %>
+        </li>
+        <li>
+          <%= t(".details.bullet2") %>
+          <br/>
+          <%= t(".details.bullet3") %>
+        </li>
```

Comment:
`bullet2a` and `bullet2b` may be clear - as both current 2 and 3 are part of the same "bullet" `<li>` item.